typhos/cache.py

```diff
@@ -210,3 +225,120 @@ def get(self, obj):
             if desc is not None:
                 # In certain scenarios (such as testing) this might happen
                 self._new_description(obj, desc)
+
+
+# The default stale cached_path threshold time, in seconds:
+TYPHOS_PATH_CACHE_TIME = int(os.environ.get('TYPHOS_PATH_CACHE_TIME', '600'))
```

Comment:
Maybe we should switch this envvar name to be explicit about the PATH being DISPLAYS_PATH?

Reply:
I'm happy to change it if you can suggest a name without making it extremely long to type?

Reply:
I was going to suggest the addition of displays in the name but it will be long... feel free to skip this one here.
`TYPHOS_DISPLAYS_PATH_CACHE_TIME` is ginormous

Reply:
Tie breaker @ZLLentz?

Reply:
I think it's better to be clear than succinct (of course being both of these is best, but...)

`TYPHOS_DISPLAY_PATH_CACHE_TIME` is my vote, but I don't have a strong opinion here since I don't expect this to be changed often.

Reply:
Resolved in 487f98e